### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by junman95**
# Pull Request Template

## Type of PR

- [ ] Feature - Platform
- [ ] Feature - Game
- [ ] Fix - Bug
- [x] docs

## Description of PR
- create rules and support file for contributors.

## Checklist

- [ ] Prepared for code review (Assignees set, Labels applied, etc.)
- [ ] Checked that changes do not negatively affect existing features
- [x] Documentation completed for new features
- [ ] All unit tests passed
- [ ] Testing in the test environment completed
- [ ] Linked this PR in the related issue
- [x] Communicated sufficiently with the relevant team or person before submitting PR

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,89 @@
 # How to contribute
-WIP
+
+## Befor start contribution
```
Comment (Ooyd) [inline docs/CONTRIBUTING.md]: Improved to : "before"
Reason: misspelling

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
...
+### You should know few terms
```
Comment (Ooyd) [inline docs/CONTRIBUTING.md]: Improved to : "You Should Know a Few Terms"
Reason : Adding "a" before "few" is grammatically correct in English

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,2 +1,89 @@
...
+### Declare meta data
+you should declare your file meta data in `src > game > meta-data.ts`
+```
+{
+  id: "my-game", // identifier to locate resource path
+  title: "MyGame", // How your game's name reveals
+  description: "My fancy js game", // description
+  author: "Junman Choi", // author
+  root: "index.html", // root html file located in your game resource directory(recommend to position on root of your game dir)
+  icon: { // icon will apperr on side bar
```
Comment (Ooyd) [inline docs/CONTRIBUTING.md]: Improved to : icon will appear on the side bar
Reason: misspelling

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (junman95) [reply docs/CONTRIBUTING.md]: thx, it's done

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (junman95) [reply docs/CONTRIBUTING.md]: thx, it's done

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (junman95) [reply docs/CONTRIBUTING.md]: thx, it's done

**Review by XPECTER — APPROVED** [review]

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (smshack) [reply docs/CONTRIBUTING.md]: good code

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (smshack) [reply docs/CONTRIBUTING.md]: great job

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (smshack) [reply docs/CONTRIBUTING.md]: good man thank you

**Comment by smshack** [comment]
good job man

**Review by smshack — APPROVED** [review]
I will proceed by referring to this good document carefully.

**Pull request merged**